### PR TITLE
MINOR: Support --release in FeatureCommand

### PR DESCRIPTION
**Pull request by dengziming**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

Support --release in FeatureCommand, currently, we only have one feature which is metadata.version.

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

Added a test case in FeatureCommandTest

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by Kvicii — APPROVED** [review]

**Comment by dengziming** [comment]
cc @mumrah

File: core/src/test/scala/unit/kafka/admin/FeatureCommandTest.scala [hunk 1]
```diff
@@ -107,4 +116,17 @@ class FeatureCommandTest extends BaseRequestTest {
...
     }
   }
+
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("kraft"))
+  def testUpdateFeatureByReleaseVersion(quorum: String): Unit = {
+    val initialDescribeOutput = TestUtils.grabConsoleOutput(FeatureCommand.mainNoExit(Array("--bootstrap-server", bootstrapServers(), "describe")))
+    assertTrue(initialDescribeOutput.contains("Feature: metadata.version\tSupportedMinVersion: 1\tSupportedMaxVersion: 7\tFinalizedVersionLevel: 1"))
+
+    FeatureCommand.mainNoExit(Array("--bootstrap-server", bootstrapServers(), "upgrade", "--release", IBP_3_3_IV3.version()))
```
Comment (mumrah) [inline core/src/test/scala/unit/kafka/admin/FeatureCommandTest.scala:126]: Can we add a negative test case where `--release` and `--feature` are given?

**Review by mumrah — COMMENTED** [review]
Thanks @dengziming! Can you also add support for `--release` to the describe sub-command?

File: core/src/test/scala/unit/kafka/admin/FeatureCommandTest.scala (hunk 1 above)
Comment (dengziming) [reply core/src/test/scala/unit/kafka/admin/FeatureCommandTest.scala:126]: I moved the original `FeatureCommandTest` to `FeatureCommandIntegrationTest` and add a new `FeatureCommandTest` to do these unit tests.

**Comment by dengziming** [comment]
@mumrah Thank you for your reminder, I added `--release` support for describing and added a unit test for it.

**Comment by dengziming** [comment]
@mumrah Do we still need --release after we add --metadata? I will close this if it's unnecessary or rebase this PR onto trunk if it's still useful.

**Comment by dengziming** [comment]
Closing this since we are using --metadata instead of --release.

**Pull request closed without merging**